### PR TITLE
Don't assume toolbar is present

### LINKED ISSUE #4537: CMS_TOOLBAR_HIDE causes 'WSGIRequest' object has no attribute 'toolbar'
When `CMS_TOOLBAR_HIDE = True` then saving plugin causes `'WSGIRequest' object has no attribute 'toolbar'` `AttributeError` in `pageadmin.py`.

In `pageadmin.py`this:

```py
if not request.toolbar
```

should probably be replaced with:

```py
if not getattr(request, 'toolbar', False)
```

### PR DESCRIPTION
**Pull request by mkoistinen**
Fixes #4537 for me.

### CODE REVIEW

File: docs/reference/configuration.rst [hunk 1]
```diff
@@ -922,7 +922,8 @@ CMS_TOOLBAR_HIDE
 default
     ``False``
 
-If True, the toolbar is hidden in the pages out django CMS.
+If True, the toolbar is hidden, even on CMS pages.
```
Comment (evildmp) [inline docs/reference/configuration.rst]: Use double backticks around literals - ``True``

**Comment by evildmp** [comment]
Shouldn't this include a regression test?

**Comment by mkoistinen** [comment]
Well, we have tests for the toolbar working in general for both states of the setting `CMS_TOOLBAR_HIDE`, and we have tests for creating and saving plugins. All of which pass, so, I think we're ok.

**Comment by yakky** [comment]
:+1:

**Pull request merged**